src/Core_Command.php

```diff
@@ -158,6 +161,12 @@ public function download( $args, $assoc_args ) {
 			$download_url = str_replace( '.zip', '.tar.gz', $offer['download'] );
 		}
 
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) ) {
+			$offer = unserialize( self::_read( 'https://api.wordpress.org/core/version-check/1.6/' ) );
+			$download_url = $offer['offers'][0]["packages"]["no_content"];
+			$version = $offer['offers'][0]["current"].' without content';
+		}
```

Comment:
Since there's a `Nightly builds are only available for the en_US locale.` error, packages with no content are also only available for the en_US locale.

Reply:
Thanks @ocean90, I've just added this https://github.com/AaronRutley/core-command/commit/dd9a40ca9b6f6cdf54329c704ad9bcf23791934d